src/referencePackageSourceGenerator/Directory.Build.props

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <PropertyGroup>
+    <ImportNetSdkFromRepoToolset>false</ImportNetSdkFromRepoToolset>
+    <!-- <ExcludeRestorePackageImports>true</ExcludeRestorePackageImports> -->
```

Comment:
What's the value in this commented out property?

Reply:
This is not needed.  I removed it.